packages/components/addon/components/hds/tabs/index.hbs

```diff
@@ -2,68 +2,25 @@
 {{! template-lint-disable require-context-role }}
 
 <div class={{this.classNames}} ...attributes>
-  {{!-- {{yield}} --}}
-
   {{! Hds-TabList }}
```

Comment:
Probably (?) eventually I'll need to separate out the Tab(s)List as another child component but not yet for implementing the initial basic functionality.